docs/framework.md

```diff
@@ -1458,55 +1276,39 @@ P   U
 
 </div>
 
-Information on the eligibility criteria for bidders can be provided using `tender` section.
-
-**Schema**: Information can be provided using the following OCDS fields.
-
-```eval_rst
-.. jsonschema:: _static/patched/release-schema.json
-    :include: tender/eligibilityCriteria
-    :collapse:
-    :nocrossref:
-```
-
-**JSON example:**
+Use [Document](reference/documents) objects in the [tender/documents](../reference/schema/#release-schema.json,,tender/documents) array to provide links to procurement documents, feasibility studies, including land acquisition, social, environmental, and rehabilitation related information and reports of independent procurement auditors. A short summary of each document can be provided using its `description` field. Each document's `documentType` field should be set to a value from the [document type codelist](../reference/codelists/#documenttype), to identify the type of document being disclosed.
 
-```eval_rst
-.. jsoninclude:: examples/full.json
-  :jsonpointer: /releases/1/tender
-  :include_only: eligibilityCriteria
+**Example**: See [section I.4](#i-4-project-economic-and-social-benefits) for JSON and flattened examples of the `documents` building block.
 
-```
+### II.2. List of pre-qualified suppliers
 
-**Flattened example** (showing top-level fields only)
+Pre-qualification or shortlist.
```

Comment:
Should this be:

```suggestion
> Pre-qualification or shortlist.
```

Reply:
I don't think so. The names from the framework provided under other headings aren't quoted. Only definitions from other sources are quoted. However, perhaps we could make it clearer that the names are taken directly from the framework.

Reply:
I think it would be clearest for them all to be indicated somehow. I think quoting is fine for that purpose.

Reply:
In https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/ef0651de03805ab5f86c3493e16fb4452b0422ae I've quoted all definitions from the framework and removed any which simply repeat the headings.

Reply:
I think maybe this specific one isn't quoted yet.